addons/elk-uninstall.yml

```diff
@@ -0,0 +1,42 @@
+# Remove the ELK stack. 
+
+# CHECK SECURITY - when customizing you should leave this in. If you take it out
+# and forget to specify security.yml, security could be turned off on components
+# in your cluster!
+
+- hosts: role=worker
+
+  vars:
+    elasticsearch_remove_data: false
+
+  tasks:
+
+    - include_vars: ../roles/elk/defaults/main.yml
```

Comment:
hm, I'm not sure how we'd know if they used one though. but you should just be able to pass one in with `-e`